### PR TITLE
[IMP] Add Françis Laffineur in demo data for HR

### PR DESCRIPTION
**Pull request by nseinlet**
@odony Improve hr demo data

### CODE REVIEW

File: addons/hr/data/hr_demo.xml [hunk 1]
```diff
@@ -308,6 +308,16 @@ If you have development competencies, we can propose you specific traineeships</
           <field name="work_email">jep@openerp.com</field>
           <field name="image" type="base64" file="hr/static/img/employee_jep-image.jpg"/>
       </record>
+      
+      <record id="employee_fla" model="hr.employee">
+          <field name="name">Françis Laffineur</field>
```
Comment (guewen) [inline addons/hr/data/hr_demo.xml]: shouldn't it be spelled `Francis`?

**Comment by dreispt** [comment]
@tde-banana-odoo Come on! What's wrong with the glasses! 😺

**Comment by tde-banana-odoo** [comment]
@dreispt Mmh ... ok for those

beer-glass-range

**Comment by dreispt** [comment]
LOL

**Comment by mart-e** [comment]
Sorry but for performances reason, we can not accept Francis.

**Pull request closed without merging**